src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/EnvelopeMessager.kt

```diff
@@ -21,7 +22,7 @@ private val logger = LoggerFactory.getLogger(EnvelopeMessager::class.java)
 class EnvelopeMessager( @Autowired client: QueueClient) : IQueueClient by client{
 
     @Throws(JSONException::class, ServiceBusException::class, InterruptedException::class)
-    fun sendMessageFromFile(jsonFileName: String, caseRef: String?, poBox: UUID?) {
+    fun sendMessageFromFile(jsonFileName: String, caseRef: String?, poBox: UUID?,  documentUrl: String) {
```

Comment:
too many spaces

Reply:
perhaps `documentUrl` used to override json content could be optional (same as `caseRef` and `poBox`)?

Reply:
sure